frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableRowExperiment.tsx

```diff
@@ -1,31 +1,45 @@
 import React from 'react';
-import { Skeleton } from '@patternfly/react-core';
-import { Td } from '@patternfly/react-table';
+import { Skeleton, Tooltip } from '@patternfly/react-core';
 import { Link } from 'react-router-dom';
 import { usePipelinesAPI } from '~/concepts/pipelines/context';
 import { experimentRunsRoute } from '~/routes';
 import { ExperimentKF } from '~/concepts/pipelines/kfTypes';
+import { NoRunContent } from '~/concepts/pipelines/content/tables/renderUtils';
 
 type PipelineRunTableRowExperimentProps = {
-  experiment: ExperimentKF | null;
+  displayName?: string;
+  experiment?: ExperimentKF | null;
   loaded: boolean;
+  error?: Error;
 };
 
 const PipelineRunTableRowExperiment: React.FC<PipelineRunTableRowExperimentProps> = ({
+  displayName,
   experiment,
   loaded,
+  error,
 }) => {
   const { namespace } = usePipelinesAPI();
+
+  if (error) {
+    return (
+      <Tooltip content={error.message} position="right">
+        <div className="pf-v5-u-disabled-color-100 pf-v5-c-truncate__start">{displayName}</div>
```

Comment:
whats with the class names? are they needed

Reply:
Hmm, good question. I just copied what it has in `PipelineVersionLink`, I will double-check and see if I can remove it.